### PR TITLE
docs/quick-start: update application

### PR DESCRIPTION
**Pull request by ilgooz**

### CODE REVIEW

**Comment by NicolasMahe** [comment]
Let's wait for mesg-js to be release as v2 before merge!

File: docs/guide/start-here/quick-start-guide.md [hunk 1]
```diff
@@ -57,17 +57,32 @@ npm init && npm install --save mesg-js
 Now, let's create an `index.js` file and with the following code:
 
 ```javascript
-const MESG = require('mesg-js').application()
+import { application, CoreTypes } from 'mesg-js'
```
Comment (NicolasMahe) [inline docs/guide/start-here/quick-start-guide.md]: are you sure about `import` `from` in nodejs? shouldn't it be `require`?

File: docs/guide/start-here/quick-start-guide.md [hunk 2]
```diff
@@ -57,17 +57,32 @@ npm init && npm install --save mesg-js
...
 const webhook    = '__ID_SERVICE_WEBHOOK__' // To replace by the Service ID of the Webhook service
 const invitation = '__ID_SERVICE_INVITATION_DISCORD__' // To replace by the Service ID of the Invite Discord service
 const email      = '__YOUR_EMAIL_HERE__' // To replace by your email
 const sendgridAPIKey = '__SENDGRID_API_KEY__' // To replace by your SendGrid API key. See https://app.sendgrid.com/settings/api_keys
 
-MESG.whenEvent(
-  { serviceID: webhook, eventKey: 'request' },
-  { serviceID: invitation, taskKey: 'send', inputs: { email, sendgridAPIKey } },
-)
+var listenReq = new CoreTypes.ListenEventRequest()
```
Comment (NicolasMahe) [inline docs/guide/start-here/quick-start-guide.md]: Can you update. this is with auto-generated code

**Comment by ilgooz** [comment]
I'm closing this one, it's already included to https://github.com/mesg-foundation/core/pull/698.

**Pull request closed without merging**